spec/coercer_spec.rb

```diff
@@ -14,15 +14,41 @@
           assert_nil(coercer.coerce_string(nil, :allow_nil => true))
         end
 
-        it "when empty string returns empty string" do
-          assert_equal("", coercer.coerce_string("", :allow_nil => true))
+        it "when empty string returns nil" do
+          assert_nil(coercer.coerce_string("", :allow_nil => true))
         end
 
         it "when false converts to string" do
           assert_equal("", coercer.coerce_string(nil, :allow_nil => false))
         end
       end
 
+      describe ":nil_when_empty" do
```

Comment:
Now if anything starts failing in our codebase we can bandaid it by adding :nil_when_empty => false to whatever attribute is causing issues